Source/UI/Controllers/CardViewController.swift

```diff
@@ -179,8 +178,7 @@ public class CardViewController: UIViewController,
 
     @objc func onTapAddressView() {
         navigationController?.pushViewController(addressViewController, animated: true)
-        // add  let logger: FramesEventLogging to checkoutAPIService
-        //checkoutApiClient?.logger.log(.billingFormPresented)
```

Comment:
no need for this line. 
` //checkoutApiClient?.logger.log(.billingFormPresented)`